operator/src/main/java/oracle/kubernetes/operator/helpers/JobHelper.java

```diff
@@ -305,10 +315,6 @@ private String getAsServiceName() {
           addEnvVar(vars, "ADMIN_PORT_SECURE", "true");
         }
         addEnvVar(vars, "AS_SERVICE_NAME", getAsServiceName());
-
-        // TBD Tom Barnes, Johnny Shum
```

Comment:
I think Tom added that for a reason. At one point in early stage of MII,  I use it for dynamic update - i.e. the job pod will use wlst online to update configuration.  That's one reason I can think of.  util.sh also has a function using it.

Reply:
I have the same recollection - and agree that the comment seems outdated now.  
Please resolve the comment if you're OK with it's removal.